test/testUtils.js

```diff
@@ -1,11 +1,20 @@
 var request = require('supertest');
 var nock = require('nock');
 var assert = require('assert');
+var redis = require('../redis.js');
 
 afterEach(function() {
   assert(nock.isDone(), 'All requests have been made. Pending: ' + nock.pendingMocks());
 });
 
+before(function() {
+  return redis.select(5);
```

Comment:
Have you just picked the number?
Shouldn't we set this up to 1001 or something similar?
Just in case someone uses a normal number ...